ui/widgets/src/aptoide/java/com/appcoins/wallet/ui/widgets/GamesBundleComposable.kt

```diff
@@ -0,0 +1,196 @@
+package com.appcoins.wallet.ui.widgets
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat.startActivity
+import coil.compose.AsyncImage
+import com.appcoins.wallet.ui.common.theme.WalletColors
+
+
+@Composable
+fun GamesBundle(
+  items: List<GameData>,
+  fetchFromApiCallback: () -> Unit
+) {
+  fetchFromApiCallback()
```

Comment:
Shouldn’t it be on the viewModel? why are you calling it here and not before?

Reply:
The call for the fetchFromApiCallback can only be called on the Aptoide version of the wallet, and not on the Google Play version. So it needs to be inside the file GamesBundleComposable.kt from the Aptoide's Build Variants.